# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieNotification.cpp
# Commit: 6379a4984eba
# Full Hash: 6379a4984ebafb659428ec3e4493f72173b3a70a
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-22 04:22:03
# Description:
#   Bug 1854028 - Update BounceTrackingStorageObserver cookie check and remove nsICookieNotification isThirdPartyCookie. r=bvandersloot,cookie-reviewers,anti-tracking-reviewers,timhuang
#   
#   For the BounceTrackingStorageObserver, instead of passing in a third-party flag
#   we can simply check if the changed cookie is partitioned or not.
#   
# ==============================================================================

diff -r 24c5f04d3793 -r 6379a4984eba netwerk/cookie/CookieNotification.cpp
--- a/netwerk/cookie/CookieNotification.cpp	Thu Sep 21 14:30:17 2023 +0000
+++ b/netwerk/cookie/CookieNotification.cpp	Thu Sep 21 14:42:32 2023 +0000
@@ -55,12 +55,6 @@
   return NS_OK;
 }
 
-NS_IMETHODIMP
-CookieNotification::GetIsThirdPartyCookie(bool* aResult) {
-  *aResult = mIsThirdPartyCookie;
-  return NS_OK;
-}
-
 NS_IMPL_ISUPPORTS(CookieNotification, nsICookieNotification)
 
 }  // namespace mozilla::net